src/mac_platform.m

```diff
@@ -2,7 +2,7 @@
 
 static NSWindow* window;
 
-bool running = true; // time to signal exit from main loop
+extern bool running;  // <- declare, but do not define
```

Comment:
Needed to declare extern to properly communicate state back to main loop on Mac platform, but more importantly this appears to be the proper variable declaration for how 'running' is used across multiple files.